src/libpriv/rpmostree-core.c

```diff
@@ -727,6 +968,47 @@ append_quoted (GString *r, const char *value)
     }
 }
 
+/* translates cachebranch back to nevra */
+static char *
+cache_branch_to_nevra (const char *cachebranch)
```

Comment:
This is the type of thing that wants inline unit tests.  Not necessarily something to do in this patch, just noting in passing.